soccer/gameplay/root_play.py

```diff
@@ -192,7 +195,7 @@ def goalie_id(self):
 
     @goalie_id.setter
     def goalie_id(self, value):
-        self._goalie_id = None if value == -1 else value
+        self._goalie_id = value
```

Comment:
Is there a reason these checks were removed? I can't find any use of this method, so I'm not sure if it matters or not.